package.json

```diff
@@ -27,8 +27,8 @@
     "test-plugins": "./bin/plugin/cli.js test-plugins",
     "test-plugins-multisite": "./bin/plugin/cli.js test-plugins --sitetype=multi",
     "enabled-modules": "./bin/plugin/cli.js enabled-modules",
-    "format-js": "wp-scripts format ./bin",
-    "lint-js": "wp-scripts lint-js ./bin",
+    "format-js": "wp-scripts format",
```

Comment:
This is a good catch. Interesting that these were previously only running against the `./bin` directory. I wonder if there was a reason for that limitation previously?

Reply:
I think it's because there were no JS files elsewhere in the project before.

Reply:
Yes, for a long time the only JS in the repo were the workflow files. Makes sense to loosen that now.